gunicorn_config.py

```diff
@@ -10,15 +10,18 @@
 bind = "0.0.0.0:{}".format(os.getenv("PORT"))
 accesslog = '-'
 
-# See AWS doc
+# To avoid load balancers reporting errors on shutdown instances, see AWS doc
 # > We also recommend that you configure the idle timeout of your application
-# to be larger than the idle timeout configured for the load balancer.
+# > to be larger than the idle timeout configured for the load balancer.
 # > By default, Elastic Load Balancing sets the idle timeout value for your load balancer to 60 seconds.
 # https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html#connection-idle-timeout
 in_production = os.environ.get("NOTIFY_ENVIRONMENT", "") == "production"
 if in_production:
     keepalive = 75
 
```

Comment:
Unsure if I had to use the `in_production` condition on the new configuration. I would like that to apply on staging but there is no such configuration environment (what do you think?). Does `testing` applies for staging?

Reply:
`NOTIFY_ENVIRONMENT` is currently set to `production` in staging. That's could be misleading (or could be explained by the fact that you want to match really production conditions), anyway at the moment this is how it is and you can expect it to work this way.

Reply:
Thanks for the confirmation.